samples/monster_generated.h

```diff
@@ -294,6 +294,19 @@ inline flatbuffers::Offset<Monster> CreateMonster(flatbuffers::FlatBufferBuilder
     _o->equipped.Pack(_fbb));
 }
 
+}  // namespace Sample
+}  // namespace MyGame
+
+namespace flatbuffers {
```

Comment:
Hmm, that's a shame it has to be in the flatbuffers namespace, but I guess there's no other way.